src/dotnetreleases/Directory.Build.props

```diff
@@ -0,0 +1,290 @@
+<Project>
```

Comment:
I originally had the "-", but then ran into issues with the subset processing so I ended removing it. What if we did an "_" instead, so dotnet_releases?

Given that you can run commands like ```build -subset clickonce+dotnetreleases``` I figured that "-" might be used to exclude subsets from the default list.

Reply:
I tried using DotNetReleases, but it seems to get converted to lowercase and then subset validation appears to be using case sensitive compares. We can't use "." separators because that is already used for subsets of subsets. I'm going to go with my original plan and just use "_"

Reply:
I pushed a compromise. I renamed the source folder to src\DotNetReleases, making it a bit more readable.